common/src/main/java/org/apache/falcon/security/AuthenticationInitializationService.java

```diff
@@ -83,8 +83,13 @@ public void init() throws FalconException {
             String authTokenValidity = StartupProperties.get().getProperty(AUTH_TOKEN_VALIDITY_SECONDS);
             long validateFrequency;
             try {
+                // -100 so that revalidation is done before expiry.
                 validateFrequency = (StringUtils.isNotEmpty(authTokenValidity))
-                        ? Long.parseLong(authTokenValidity) : 86400;
+                        ? (Long.parseLong(authTokenValidity) - 100) : 86300;
```

Comment:
minor nit: Can we define a constant and say what is 86400 maps to and then -100 so that this is more readable?